pkg/e2e/operators/cloudingress/public_private.go

```diff
@@ -98,4 +112,23 @@ func updateApplicationIngress(h *helper.H, lbscheme string) {
 	Expect(err).NotTo(HaveOccurred())
 }
 
+//Function to grab the default ingresscontroller
+func appIngress(h *helper.H, isdefault bool, dnsname string) (appIngress cloudingressv1alpha1.ApplicationIngress, exists bool, index int) {
```

Comment:
I think this [function](https://github.com/openshift/osde2e/blob/main/pkg/e2e/operators/cloudingress/secondary_router.go#L74) does the same thing, so there is an opportunity to factorize